qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -52,13 +52,14 @@ class QuantumKernel:
     algorithms such as support vector classification, spectral clustering or ridge regression.
     """
 
-    @deprecate_arguments("0.5.1", {"user_parameters": "training_parameters"})
+    @deprecate_arguments("0.5.0", {"user_parameters": "training_parameters"})
     def __init__(
         self,
         feature_map: Optional[QuantumCircuit] = None,
         enforce_psd: bool = True,
         batch_size: int = 900,
         quantum_instance: Optional[Union[QuantumInstance, Backend]] = None,
+        user_parameters: Optional[Union[ParameterVector, Sequence[Parameter]]] = None,
         *,
```

Comment:
I thought the deprecate arguments decorator renamed the parameter to the new one. Hence I don't think its needed in the constructor. Any new code should be dealing with training_parameters. @manoelmarques please correct me if am wrong since you are more familiar with the decorators than I. Hence aside from the addition of the decorator, the only only change I would expect to see over the old init is just the renaming of user_parameters -> training_parameters since their typehint/function is unaltered otherwise. I think you can also drop the text about user_parameters from the docstring since this really only applies to supporting old code and if you have no old code, where you will see the deprecation messages emitted when running it (assuming you called it via keyword arg).

Reply:
Aha, right.  No need for user_parameters at all.  Been a long time since I coded Python, all this decorator stuff is new to me.  Nuked user_parameters in the constructor.